packages/nimble-components/src/table/index.ts

```diff
@@ -184,6 +185,26 @@ export class Table<
     @observable
     public showCollapseAll = false;
 
+    /**
+     * @internal
+     */
+    @observable
+    public canHaveCollapsibleRows = false;
+
+    /**
+     * @internal
+     */
+    @volatile
+    public get collapseButtonVisibility(): string {
+        if (!this.canHaveCollapsibleRows) {
+            return 'hidden';
```

Comment:
Since we are leveraging the subtilties of `visibility: visible`, `visibility: hidden`, and `display: none` could we choose a different class name? Maybe `hidden-reduced-size`?